frontera/contrib/backends/sqlalchemy/components.py

```diff
@@ -51,20 +51,20 @@ def frontier_stop(self):
     def add_seeds(self, seeds):
         for seed in seeds:
             o = self._create_page(seed)
-            self.cache[o.fingerprint] = self.session.merge(o)
+            self.cache[to_bytes(o.fingerprint)] = self.session.merge(o)
```

Comment:
I have used bytes as keys for the cache in both metadata and state. The reason is to reduce the number of conversions needed in other parts of the code.